generated/react-native-partition-based-sync/src/ItemSchema.js

```diff
@@ -0,0 +1,19 @@
+import {BSON} from 'realm';
+
+export class Item {
+  constructor({id = new BSON.ObjectId(), isComplete = false}) {
+    this.isComplete = isComplete;
+    this._id = id;
+  }
+
+  static schema = {
+    name: 'Task',
```

Comment:
```suggestion
    name: 'Item',
```

Looks like you forgot to change this code to 'Item'